public/index.html

```diff
@@ -29,7 +29,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>AltDotNg</title>
```

Comment:
Name of Company is not AltDotNg. It is Alternative Markets Limited. You can Alt instead.